src/types/aws/ap/registration.ts

```diff
@@ -121,7 +122,7 @@ export type TDocumentation = {
 };
 
 export type BankingDetails = {
-  BankStatementPDF: FileObject;
```

Comment:
`BankStatementPDF` is still used in lambda `hubspot-company-updatooor @jp-mariano

Reply:
This is also the case for `registration-update` Lambda's schema. I will update both files. 👍🏾